releases.yaml

```diff
@@ -0,0 +1,16 @@
+sources:
+  - id: keycloak
+    productName: Keycloak
+    repo: keycloak/keycloak
+    githubReleaseFrom: 23.0.0
```

Comment:
What's `githubReleaseFrom` used for? We've uploaded KC releases to GitHub releases for much longer than 23.0.0